addons/mail/static/src/models/message_seen_indicator/message_seen_indicator.js

```diff
@@ -17,6 +17,7 @@ function factory(dependencies) {
          * @param {mail.message|integer} message
          * @param {mail.thread|integer} thread
          * @returns {string|undefined}
+         * FIXME replace by using messageId & channelId as identifying fields (task-2335647)
```

Comment:
Sorry for not being clear directly, but docstring comment should be above `@` tags